concepticondata/conceptlists/Heeschen-1978-104.tsv

```diff
@@ -0,0 +1,106 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Heeschen-1978-1	1	all	98	ALL
+Heeschen-1978-2	2	ashes	646	ASH
+Heeschen-1978-3	3	bark	1204	BARK
+Heeschen-1978-4	4	belly	1251	BELLY
+Heeschen-1978-5	5	big	1202	BIG
+Heeschen-1978-6	6	bird	937	BIRD
+Heeschen-1978-7	7	let us bite	1403	BITE
+Heeschen-1978-8	8	black	163	BLACK
+Heeschen-1978-9	9	blood	946	BLOOD
+Heeschen-1978-10	10	bone	1394	BONE
+Heeschen-1978-11	11	breast	1402	BREAST
+Heeschen-1978-12	12	let us burn	2102	BURN
+Heeschen-1978-13	13	claw	72	CLAW
+Heeschen-1978-14	14	cloud	1489	CLOUD
+Heeschen-1978-15	15	cold	1287	COLD
+Heeschen-1978-16	16	come!	1446	COME
+Heeschen-1978-17	17	died	2101	DIE OR BE DEAD
+Heeschen-1978-18	18	dog	2009	DOG
+Heeschen-1978-19	19	word, language	1307	LANGUAGE
+Heeschen-1978-20	20	drink	1401	DRINK
+Heeschen-1978-21	21	dry	1398	DRY
+Heeschen-1978-22	22	ear	1247	EAR
+Heeschen-1978-23	23	earth, ground	1228	EARTH (SOIL)
+Heeschen-1978-24	24	he is eating	1336	EAT
+Heeschen-1978-25	25	egg	744	EGG
+Heeschen-1978-26	26	eye	1248	EYE
+Heeschen-1978-27	27	fat	323	FAT (ORGANIC SUBSTANCE)
+Heeschen-1978-28	28	later		
+Heeschen-1978-29	29	fire	221	FIRE
+Heeschen-1978-30	30	flying	1441	FLY (MOVE THROUGH AIR)
+Heeschen-1978-31	31	foot	1301	FOOT
+Heeschen-1978-32	32	full	1429	FULL
+Heeschen-1978-33	33	give me	1447	GIVE
+Heeschen-1978-34	34	good	1035	GOOD
+Heeschen-1978-35	35	green, unripe	1958	UNRIPE
+Heeschen-1978-36	36	hair	1040	HAIR
+Heeschen-1978-37	37	hand	1277	HAND
+Heeschen-1978-38	38	head	1256	HEAD
+Heeschen-1978-39	39	I heard	1408	HEAR
+Heeschen-1978-40	40	heart	1223	HEART
+Heeschen-1978-41	41	I	1209	I
+Heeschen-1978-42	42	kill!	1417	KILL
+Heeschen-1978-43	43	I killed	1417	KILL
+Heeschen-1978-44	44	knee	371	KNEE
+Heeschen-1978-45	45	I know	1410	KNOW (SOMETHING)
+Heeschen-1978-46	46	I catch	702	CATCH
+Heeschen-1978-47	47	liver	1224	LIVER
+Heeschen-1978-48	48	long (way)	1203	LONG
+Heeschen-1978-49	49	louse	1392	LOUSE
+Heeschen-1978-50	50	child	1304	CHILD (YOUNG HUMAN)
+Heeschen-1978-51	51	short	1645	SHORT
+Heeschen-1978-52	52	meat	634	MEAT
+Heeschen-1978-53	53	moon	1313	MOON
+Heeschen-1978-54	54	mountain	639	MOUNTAIN
+Heeschen-1978-55	55	shoulder	1482	SHOULDER
+Heeschen-1978-56	56	name	1405	NAME
+Heeschen-1978-57	57	back	1291	BACK
+Heeschen-1978-58	58	new	1231	NEW
+Heeschen-1978-59	59	night	1233	NIGHT
+Heeschen-1978-60	60	nose	1221	NOSE
+Heeschen-1978-61	61	not	1240	NOT
+Heeschen-1978-62	62	one	1493	ONE
+Heeschen-1978-63	63	people	789	PEOPLE
+Heeschen-1978-64	64	rain	1253	RAIN (RAINING)
```

Comment:
should be [rain (precipitation)](http://concepticon.clld.org/parameters/658), as this is clearly a known, since all verbs have first person in gloss, right?